.gitignore

```diff
@@ -25,6 +25,7 @@ yarn-debug.log*
 yarn-error.log*
 
 # local env files
+.env
```

Comment:
```suggestion
```

Reply:
You should include your local env variables in .env.local since it's the convection for next.js